### PR TITLE
Enhanced UI of profile page

### PR DESCRIPTION
**Pull request by yogeshbhutkar**
Addresses issue #66 

Changes Implemented:
 - Used <i> tag instead of FontAwesome Library.
 
 Screenshot of changes:
 
 
image

### CODE REVIEW

**Review by skks1212 — CHANGES_REQUESTED** [review]
LGTM. Please use a button instead of a div and add `cursor-pointer` to the button. Also resolve merge conflicts

**Comment by skks1212** [comment]
@yogeshbhutkar are you working on this?

**Review by skks1212 — APPROVED** [review]

**Comment by skks1212** [comment]
@yogeshbhutkar please resolve merge conflicts

**Review by skks1212 — APPROVED** [review]

File: src/components/ui/interactive.tsx [hunk 1]
```diff
@@ -24,8 +24,8 @@ export function Input(
           )}
           disabled={loading}
         />
-        {right}
       </div>
+      {right}
```
Comment (Ashesh3) [inline src/components/ui/interactive.tsx:28]: @yogeshbhutkar Why was this moved down outside?

**Pull request merged**